package.json

```diff
@@ -165,6 +165,13 @@
         "width": 3,
         "workspace": "1. GSPS - Ogólne"
       },
+      {
+        "name": "stream-assign",
+        "title": "Przypisywanie streamów",
+        "file": "stream-assign.html",
+        "width": 3,
+        "workspace": "1. GSPS - Ogólne"
```

Comment:
Maybe we should move the restream specific stuff into it's own workspace so it doesn't get mixed with all the other panels?

Reply:
moved to new tab 
![image](https://github.com/user-attachments/assets/5a2db9c5-8e4a-4219-90cc-5bb1291ee82a)